### DISCUSSION TITLE
Quickstart install procedure 2.0 (SpaceOne initialization)

### DISCUSSION THREAD

**Discussion by Cclleemm** [cloudforet-io/community#148] [Bug Report]
Hello,
I have a suggestion for your quickstart documentation (on Minikube for the v2) because I can't deal with the part 5) Default Initialization (in spacectl POD) section.

You change the way to initialized spaceone (https://github.com/cloudforet-io/cloudforet-io.github.io/commit/a03adcb663575e8d4ee72ddbdfe65d64fde05099#diff-24a3379e185f0ea2913321a4983425e14b4eb3cddba4e7bc2500c1b5ed3a826bR154) but I can't understand the link between the step : 
`spacectl config init -f default.yaml`
and
`root domain yaml file (root.yaml)`

The root.yaml is not created automatilly, if we have to create it manually can you specified it in the doc ? 

However I tried to create it with this payload 

```---
admin:
    user_id: admin@example.com
    password: Admin123!@#
    name: Admin
```
But when I run `spacectl exec init identity.System -f root.yaml` I have the following error message

![image](https://github.com/user-attachments/assets/1001609e-9dea-4e2b-884a-c4ff7d3d4908)

**Comment by choonho** [comment]
In Step5, you have to execute command in spacectl POD.
When you type "kubectl get pod -n cloudforet". There is a POD as spacectl-xxxxxxxxxxxxx.

Log-in by "kubectl exec -ti spacectl-xxxxxxxxx -n cloudforet -- /bin/bash"
